README.md

```diff
@@ -123,8 +123,54 @@ public static void StreamsTrigger(
 }
 ```
 
-## Known Issues
-- The `RedisPubSubTrigger` is not capable of listening to [keyspace notifications](https://redis.io/docs/manual/keyspace-notifications/) on clustered caches.
+### `Redis` Input Binding
+
+#### Inputs
+- `ConnectionStringSetting`: Name of the setting in the appsettings that holds the to the Redis cache connection string (eg `<cacheName>.redis.cache.windows.net:6380,password=...`).
+- `Command`: The redis-cli command to be executed on the cache with all arguments separated by spaces.
+
+> **Note**
+> Not all commands are supported for this binding. At the moment, only read commands that return a single output are supported.
```

Comment:
Can we list a few more examples of what type of commands are supported and what types are not? That will help people have a better understanding. Once we've settled on a set of commands that we feel good about, we should list them all.